crates/yas_core/src/tests/test_contracts/test_yas_pool.cairo

```diff
@@ -486,81 +491,1014 @@ mod YASPoolTests {
+                            max_tick,
+                            BigNumber
+                        );
+
+                    assert(
+                        token_0.balanceOf(yas_pool.contract_address) == 9996 + 828011525,
+                        'wrong token_0 amount'
+                    );
+                    assert(
+                        token_1.balanceOf(yas_pool.contract_address) == 1000, 'wrong token_1 amount'
+                    );
+                }
 
-                    let balance_token_0 = token_0.balanceOf(yas_pool.contract_address);
-                    let balance_token_1 = token_1.balanceOf(yas_pool.contract_address);
 
-                    assert(balance_token_0 == 2000000000000000000, 'wrong balance token 0');
-                    assert(balance_token_1 == 2000000000000000000, 'wrong balance token 1');
+                #[test]
+                #[available_gas(200000000000)]
+                fn test_max_tick() {
+                    let (yas_pool, token_0, token_1, mint_callback, min_tick, max_tick) = setup();
+
+                    mint_callback
+                        .mint(
+                            yas_pool.contract_address,
+                            WALLET(),
+                            IntegerTrait::<i32>::new(22980, true),
+                            max_tick,
+                            10000
+                        );
+                    assert(
+                        token_0.balanceOf(yas_pool.contract_address) == 9996 + 31549,
+                        'wrong token_0 amount'
+                    );
+                    assert(
+                        token_1.balanceOf(yas_pool.contract_address) == 1000, 'wrong token_1 amount'
+                    );
+                }
+
+
+                // WIP: 'removing works'
+                // no burn() function
+                //#[test]
+                //#[available_gas(200000000)]
+                fn test_burn() {
+                    //to implement:
+                    //await mint(wallet.address, -240, 0, 10000) -done
+                    //await pool.burn(-240, 0, 10000)
+                    //const { amount0, amount1 } = await pool.callStatic.collect(wallet.address, -240, 0, MaxUint128, MaxUint128)
+                    //expect(amount0, 'amount0').to.eq(120)
+                    //expect(amount1, 'amount1').to.eq(0)
+
+                    let (yas_pool, token_0, token_1, mint_callback, min_tick, max_tick) = setup();
+                    mint_callback
+                        .mint(
+                            yas_pool.contract_address,
+                            WALLET(),
+                            IntegerTrait::<i32>::new(240, true),
+                            IntegerTrait::<i32>::new(0, false),
+                            10000
+                        );
+                    //yas_pool.burn(-240, 0, 10000)
+                    assert(1 == 2, 'burn() function not created');
+                }
```

Comment:
It would be better if we just comment the whole function because it's not testing anything yet.

Reply:
cool